### DISCUSSION TITLE
Can't get resource in notebooks/pySCENIC - Integration with scanpy.ipynb

### DISCUSSION THREAD

**Discussion by execbbbr** [aertslab/SCENIC#426] [Python]
Hi：

Recently, when I was studying this library in the `notebooks/pySCENIC - Integration with scanpy.ipynb`, I found the following code:

```python
# Downloaded from Cell Journal website on 1 MAR 2019.
# https://ars-els-cdn-com.kuleuven.ezproxy.kuleuven.be/content/image/1-s2.0-S0092867418311784-mmc1.xlsx
SAMPLE_METADATA_FNAME = os.path.join(BASE_FOLDER, "1-s2.0-S0092867418311784-mmc1.xlsx")
```

The mentioned file cannot be downloaded as it requires login credentials from KU Leuven University. Is it possible to provide an alternative way to download this file?

**Comment by kandabarau** [comment]
> Hi：
> 
> Recently, when I was studying this library in the `notebooks/pySCENIC - Integration with scanpy.ipynb`, I found the following code:
> 
> ```python
> # Downloaded from Cell Journal website on 1 MAR 2019.
> # https://ars-els-cdn-com.kuleuven.ezproxy.kuleuven.be/content/image/1-s2.0-S0092867418311784-mmc1.xlsx
> SAMPLE_METADATA_FNAME = os.path.join(BASE_FOLDER, "1-s2.0-S0092867418311784-mmc1.xlsx")
> ```
> 
> The mentioned file cannot be downloaded as it requires login credentials from KU Leuven University. Is it possible to provide an alternative way to download this file?

hi @execbbbr 

[that one](https://ars.els-cdn.com/content/image/1-s2.0-S0092867418311784-mmc1.xlsx) from the original [publication](https://doi.org/10.1016/j.cell.2018.09.006) is working:



bests,
Sergey

**Reply by ghuls** [reply]
Link fixed in notebook.
